cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py

```diff
@@ -5,6 +5,8 @@
 # todo: add CC3D logo to window title icon
 # todo: disable built-in key commands for closing render windows
 # todo: add support for additional plots (e.g., tracking fields)
+# todo: improve synchronization to allow asynchronous execution and visualization while handling race conditions
+# todo: implement setting a frame by provided screenshot data
```

Comment:
There is a quote from Bob Martin that I love because I am very guilty of it. It goes, "TODO means 'Don't do.'" We lose them or forget them when they're in the code. Could you create Issues for these?